components/Relationships/Relationships.tsx

```diff
@@ -17,14 +17,10 @@ const Relationships = ({ id }: Props): React.ReactElement => {
     return <ErrorMessage />;
   }
 
-  const shouldAppear =
-    (personalRelationships.parents &&
-      personalRelationships.parents.length > 0) ||
-    (personalRelationships.children &&
-      personalRelationships.children.length > 0) ||
-    (personalRelationships.siblings &&
-      personalRelationships.siblings.length > 0) ||
-    (personalRelationships.other && personalRelationships.other.length > 0);
+  let shouldAppear = false;
+  personalRelationships.map((elm) => {
```

Comment:
What's `elm`?

Reply:
I will rename it to "relationship" to make it clearer